Frameworks/CoreText/CTFrame.mm

```diff
@@ -108,13 +108,14 @@ void CTFrameDraw(CTFrameRef frame, CGContextRef ctx) {
 
         CGPoint relativePosition = frame->_lineOrigins[i];
 
-        for (size_t j = 0; j < [line->_runs count]; ++j) {
-            _CTRun* curRun = [line->_runs objectAtIndex:j];
+        CFIndex len = CFArrayGetCount(line->_runs);
+        for (size_t j = 0; j < len; ++j) {
+            __CTRun* curRun = const_cast<__CTRun*>(static_cast<CTRunRef>(CFArrayGetValueAtIndex(line->_runs, j)));
```

Comment:
wait, why does this run need to be non-const? we aren't manipulating it, so it's actually _better_ that it's const.